packages/mendel-generator-prune/README.md

```diff
@@ -0,0 +1,30 @@
+# mendel-generator-prune
+
+Mendel-generator modifies references of exposed modules across bundles.
+
+In a large application with lot of files, few common files are required by most of other files in other bundles. This generator assigns unique hash to each exposed module. Instead of referring those common files by file path, they will be referred by unique hash. Hash is computed using `shasum`.
+
+## Example
+Assume that we have 2 bundles. First bundle(the main bundle) has 3 files
+
+```
+"foo.js":"1",
+"bar".js":"2",
+"foobar".js":"SOME_HASH",
+```
+
+There is a second bundle(lazy) which has its own files as well as files from main bundle as dependencies.
+
+```
+"deps": {
+    "dep1.js":"1",
+    "dep2.js":"2",
+    "foobar".js":"SOME_HASH"
+}
+```
+
+Now, even if you update other files in `main` bundle and recreate this bundle, lazy bundle will always refer to `foobar.js` by hash.
+
+#### Known issues
+
+`shasum` generates 32 character hash which does not have good compression rate with gzip. So, you will see an increase in your bundle payload.
```

Comment:
One more reason to try to remove it. Small hash might have collision, but no hash at all will not have collisions because `normalizedId` is already consistent.